cu-webteam/d8-platform
# Tabbed accordion

# Description

Fixed the tabbed content issue where only one tabbed content body was allowed to show, even if there was more than one tabbed content on a page. Example: [http://contenttest.creighton.acsitefactory.com/content-page-bc](http://contenttest.creighton.acsitefactory.com/content-page-bc)

## Related PRs

None

## Todos

- [x] Tests _tested locally, no unit tests_
- [x] Documentation _code comments in the js file, external documentation not required_

## Deploy Notes

None

## Steps to Test or Reproduce

- Pull down tabbed_accordion branch
- Clear cache on local.creighton.com
- Add two tabbed accordions to the content page, either the side tab or top tab type
- Check the content page to ensure that both tabbed accordions have one body showing at all times
- Check that the body showing is associated with the active tab
